frontend/app/controllers/TierController.scala

```diff
@@ -90,7 +90,7 @@ class TierController(
     val sub = subscriber.subscription
 
     (for {
-      country <- EitherT(memberService.country(subscriber.contact).map(\/.right))
```

Comment:
most of the scala changes are just to get it to work with new signatues in membership-common.  The intention is the changes are minimal and just get it working.